src/main/java/org/spongepowered/common/data/provider/item/stack/BookItemStackData.java

```diff
@@ -71,11 +70,10 @@ public static void register(final DataProviderRegistrator registrator) {
                             return content.generation();
                         })
                         .setAnd((h, v) -> {
-                            if (v < 0) {
+                            if (v < 0 || v > 3) {
```

Comment:
Is it actually a limitation or just that you cant craft it in vanilla gameplay?

Reply:
Actual limitation. Also transaltion keys exist only for those generation values
![image](https://github.com/SpongePowered/Sponge/assets/82652479/9bcfef9f-ee1c-4eba-9833-ea166f9f9e74)
![image](https://github.com/SpongePowered/Sponge/assets/82652479/08163893-59f5-4b52-ac0e-a893a7ff5f9d)